doc/Developing.md

```diff
@@ -211,3 +234,11 @@ Theia uses native modules and also requires Python 2.x to be installed on the sy
 
 [all-in-one packages]: https://github.com/felixrieseberg/windows-build-tools
 [bug]: https://github.com/theia-ide/theia/issues
+
+### Root privileges errors
+When trying to install with root privileges, you might encounter errors such as `cannot run in wd`.
```

Comment:
Can we reference this closed issue? https://github.com/theia-ide/theia/issues/266#issuecomment-313328699